cwlavro-tools/src/test/resources/arraysConfirm.txt

```diff
@@ -0,0 +1,15 @@
+{
+  "filesA": [
+    {
```

Comment:
Add a second array entry to really hammer home that this is for arrays. 
Maybe `/tmp/fill_me_in_a.txt` and `/tmp/fill_me_in_b.txt` and similar for directories